sunkit_pyvista/plotter.py

```diff
@@ -345,9 +357,13 @@ def _loop_through_meshes(self, mesh_block):
                 self._loop_through_meshes(block)
             else:
                 color = dict(block.field_arrays).pop('color', [None])[0]
+                print(color)
                 if not isinstance(color, str):
```

Comment:
So color could be the colormap instance it self?

Reply:
No, it's saved as a string. That's why we have the inverted cmlist dictionary

Reply:
So what could its type be if not a string?

Reply:
`np.nan` is passed if there's no colour, hence the check

Reply:
Should we just check for that instead?

Reply:
I did that at first, `np.isnan()` errors out if we pass a string to it so that's why I chose to do it this way instead

Reply:
Why would we pass a string to it?

Reply:
== comparison with np.nan doesn't work